packages/babel-plugin-jest-hoist/package.json

```diff
@@ -11,5 +11,12 @@
   },
   "license": "MIT",
   "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "dependencies": {
+    "@types/babel__traverse": "^7.0.6"
```

Comment:
Why not install this type as a dev dependency?

Reply:
It's part of the public interface